lms/djangoapps/teams/views.py

```diff
@@ -419,6 +420,9 @@ def get(self, request):
                     topic_id=topic_id
                 )
                 return Response(error, status=status.HTTP_400_BAD_REQUEST)
+
+            if course_module.teamsets_by_id[topic_id].teamset_type == TeamsetType.private_managed:
```

Comment:
Small suggestion, since we have to do this check in a couple of places now: add a property to the `TeamsetConfig` class called `is_private_managed` that returns `self.teamset_type == TeamsetType.private_managed`